### PR TITLE
Schedule msg

### PR DESCRIPTION
**Pull request by joakim-hove**
With this PR the output from the Schedule construction will be like:

**Start**

```
Processing dynamic information from
/home/hove/work/OPM/opm-tests/norne/NORNE_ATW2013.DATA line 455
```

**Every change of input file**

```
Reading from: /home/hove/work/OPM/opm-tests/norne/INCLUDE/VFP/DevNew.VFP line 9
```

Normal `DATES`processing:

```
Starting report step 4/248 at 1997-11-14 8.0 days - line 204
Processing keyword WCONHIST at line 208
Processing keyword WCONINJE at line 212
Processing keyword GECON at line 216
Complete report step 4 (17.0 days) at 1997-12-01 / 25.0 days

Starting report step 5/248 at 1997-12-01 25.0 days - line 221
Processing keyword WCONHIST at line 225
Processing keyword WCONINJE at line 230
Complete report step 5 (16.0 days) at 1997-12-17 / 41.0 days
The remaining messages from Schedule parsing will only be in the .PRT file
....
```
Currently five such blocks will be printed - after that all messages will only go to the PRT file.

### CODE REVIEW

**Comment by alfbr** [comment]
This is really good! I believe the wording of the messages are also very clear, only two suggestions for changes from me:

Starting -> Initializing

The remaining messages from Schedule parsing will only be in the .PRT file
->
Report limit reached, see PRT-file for remaining Schedule initialization.

**Comment by joakim-hove** [comment]
Update:
```
Initializing report step 4/248 at 1997-12-17 (41.0 days) - line 235
Processing keyword WELSPECS at line 269
Processing keyword COMPDAT at line 273
Processing keyword WCONHIST at line 286
Processing keyword WCONINJE at line 292
Complete report step 4 (15.0 days) at 1998-01-01 (56.0 days)

Initializing report step 5/248 at 1998-01-01 (56.0 days) - line 297
Processing keyword WCONHIST at line 301
Processing keyword WCONINJE at line 307
Processing keyword RPTSCHED at line 311
Complete report step 5 (31.0 days) at 1998-02-01 (87.0 days)
Report limit reached, see PRT-file for remaining Schedule initialization.
```

File: src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp [hunk 1]
```diff
@@ -334,37 +378,57 @@ namespace {
         */
 
         auto restart_skip = currentStep < this->m_timeMap.restart_offset();
+        ScheduleLogger logger(*this, restart_skip);
+        {
+            const auto& schedule_keyword = section.getKeyword<ParserKeywords::SCHEDULE>();
+            const auto& location = schedule_keyword.location();
+            current_file = location.filename;
+            logger(fmt::format("Processing dynamic information from\n{} line {}\n", current_file, location.lineno));
+            logger(fmt::format("Initializing report step {}/{} at {} {} days line {}",
```
Comment (bska) [inline src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp:387]: The `days` here is technically wrong for LAB units.  How much work would it be to output the unit-system dependent time unit name instead?

Comment (joakim-hove) [reply src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp:387]: IT is ONLY the string days which is wrong. I'll fix

**Comment by alfbr** [comment]
Perfect, thank you!

**Comment by joakim-hove** [comment]
OK - I have polished this a bit; ready for review and merge from my side.

**Comment by joakim-hove** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks for the updates.  This looks good to me, and will be a significant aid to analyzing errors in the input file.  I'll merge once the build check is green.

**Comment by bska** [comment]
> I'll merge once the build check is green.

Build check is green.  I'll merge into master as advertised.

**Pull request merged**